### PR TITLE
Integrating OnPremRV with AIO

### PR DESCRIPTION
**Pull request by DukeDavis12**
Integrates the OnPremRV with AIO by

    i)   Running OnPremRV and the associated Redis DB as a docker service.
         (All the associated files are in /dockerfiles folder.)

    ii)  Internally mapping the msg/20&30 series from:8080 to the:8001 port.
         (Check RelayServlet.java and mp/web.xml for the change)

    iii) Pulling in the OnPremRV dependency along with the mvn building stage

Signed-off-by: Davis Benny <davis.benny@intel.com>

### CODE REVIEW

File: container/overlay/tomcat/db/v1/defaults/svi.json [hunk 1]
```diff
@@ -31,8 +31,8 @@
     "module": "sdo_sys",
     "msg": "exec",
     "valueLen": -1,
-    "valueId": "sh-linux64",
+    "valueId": "binsh-linux64",
```
Comment (trbehera) [inline container/overlay/tomcat/db/v1/defaults/svi.json:34]: This is an unrelated change, should be a separate PR.

File: container/overlay/dockerfiles/hashlist.redis [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Supply Chain Toolkit - Manufacturer ECDSA 256 public key hash
```
Comment (trbehera) [inline container/overlay/dockerfiles/hashlist.redis:1]: Remove the comments as it may cause warnings.

File: container/overlay/tomcat/webapps/mp/WEB-INF/web.xml [hunk 3]
```diff
@@ -36,7 +36,7 @@
     <servlet-class>org.sdo.demo.RelayServlet</servlet-class>
     <init-param>
       <param-name>forward-to</param-name>
-      <param-value>rps</param-value>
+      <param-value>8001</param-value>
```
Comment (trbehera) [inline container/overlay/tomcat/webapps/mp/WEB-INF/web.xml:39]: Comment:
Use value "rps" for using PRI Rendezvous service. Use value "8001" to use On-Prem Rendezvous-Service.

File: container/overlay/dockerfiles/Dockerfile-redis [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+FROM redis:latest
+
+# Copy Files
+COPY ./dockerfiles/hashlist.redis /usr/local/
+COPY ./dockerfiles/startRedis.sh /usr/local/
+
+# Start Service
+ENTRYPOINT ["sh", "/usr/local/startRedis.sh"]
+
+# Docker image helathcheck
```
Comment (KiranSukhavasi) [inline container/overlay/dockerfiles/Dockerfile-redis:12]: spell mistake - healthcheck

File: container/overlay/dockerfiles/cmdDocker.sh [hunk 5]
```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+# Copyright 2019 Intel Corporation
```
Comment (KiranSukhavasi) [inline container/overlay/dockerfiles/cmdDocker.sh:2]: Change copyright to 2020
